templates/_components/tag.twig

```diff
@@ -33,7 +33,7 @@
 {% tag el with attrs|merge({
   href: href,
   type: clickable and el == 'button' ? 'button' : null,
-  class: classNames('inline-flex items-center justify-center gap-4 rounded-sm text-center font-medium focus:outline-none focus-visible:ring-4', {
+  class: classNames('inline-flex items-center justify-center gap-4 rounded-xs text-center font-medium focus:outline-hidden focus-visible:ring-4', {
```

Comment:
TW-4 renamed a few utilities to [make them more consistent](https://tailwindcss.com/docs/upgrade-guide#renamed-utilities). The automated migration handled most of them.